dataservice/api/participant/models.py

```diff
@@ -87,5 +110,86 @@ def immediate_relatives(self):
         return chain(self.upstream_immediate_relatives(),
                      self.downstream_immediate_relatives())
 
+    def add_alias(self, pt):
```

Comment:
Seems like this could be a constructor in an association proxy, though I wasn't able to get it working.

Reply:
Hmm yea if we had an `aliases` association proxy, then this method would be the creator for it and then I migtht be able to provide a getset_factory to return the result of my @property aliases method. I don't know what function I would use for the setter in getset_factory though. 

I'd prefer to stick with what I have for now though. It seems pretty straightforward and I've found association proxies to be a little tricky sometimes